### PR TITLE
Replace mission manager with mission control

### PR DESCRIPTION
**Pull request by patapufate**
# Description

This PR removes the mission manager package and replace it with the mission control packages
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The code builds clean without any errors or warnings
- [] Unit tests are passing
- [x] Linter tests are passing

For testing this PR depends on #117 and #116

### CODE REVIEW

File: autopilot/CMakeLists.txt [hunk 1]
```diff
@@ -8,15 +8,14 @@ add_compile_options(-std=c++11)
 find_package(catkin REQUIRED COMPONENTS
   auv_interfaces
   fin_control
-  geodesy
```
Comment (hidmic) [inline autopilot/CMakeLists.txt:11]: @patapufate hmm, removing waypoint following?

**Review by hidmic — COMMENTED** [review]
Overall LGTM but for one comment.

File: autopilot/CMakeLists.txt (hunk 1 above)
Comment (patapufate) [reply autopilot/CMakeLists.txt]: Good catch. Restored in 8261d61

Comment (hidmic) [reply autopilot/CMakeLists.txt]: I think we'll need to revert more than that -- I see lots of changes to the `autopilot`.

**Review by hidmic — APPROVED** [review]

**Pull request merged**